diff --git a/node_modules/react-native-url-preview/index.js b/node_modules/react-native-url-preview/index.js
index 3d596c9..07eafe0 100644
--- a/node_modules/react-native-url-preview/index.js
+++ b/node_modules/react-native-url-preview/index.js
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { Component } from 'react';
 import {getLinkPreview} from "link-preview-js";
 import PropTypes from "prop-types";
 import {
@@ -8,8 +8,9 @@ import {
   Text,
   TouchableOpacity,
   View,
-  ViewPropTypes
+  StyleSheet
 } from "react-native";
+import { ViewPropTypes } from 'deprecated-react-native-prop-types';
 
 const REGEX = /[-a-zA-Z0-9@:%_\+.~#?&//=]{2,256}\.[a-z]{2,4}\b(\/[-a-zA-Z0-9@:%_\+.~#?&//=]*)?/g;
 
@@ -254,8 +255,8 @@ RNUrlPreview.propTypes = {
   faviconStyle: ViewPropTypes.style,
   textContainerStyle: ViewPropTypes.style,
   title: PropTypes.bool,
-  titleStyle: Text.propTypes.style,
-  titleNumberOfLines: Text.propTypes.numberOfLines,
-  descriptionStyle: Text.propTypes.style,
-  descriptionNumberOfLines: Text.propTypes.numberOfLines
+  titleStyle: ViewPropTypes.style,
+  titleNumberOfLines: PropTypes.number,
+  descriptionStyle: ViewPropTypes.style,
+  descriptionNumberOfLines: PropTypes.number
 };
